app/assets/stylesheets/content/_tooltips.sass

```diff
@@ -182,3 +182,48 @@ table
       padding: 4px
       font-size: 0.8em
       color: #505050
+
+// tooltips with html support
+span.op_tooltip
```

Comment:
`op_tooltip` is not really a great name. Sorry for bitching, but this should be renamed. Have a look at the way the simple `tooltip`s are implemented.